Podfile.lock

```diff
@@ -60,6 +60,7 @@ PODS:
     - Reachability
     - SAMKeychain
     - SocketRocket
+    - SwiftProtobuf
```

Comment:
Aren't there corresponding changes to `BUILDING.md` around using brew to install a dependency or two of `SwiftProtobuf`?

Reply:
IMO the instructions for generating protobufs aren't really relevant to people building the app, which is why there is a separate README in the protobufs subdir (which I updated in this PR).